contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -780,116 +826,95 @@ contract NodeOperatorsRegistry is AragonApp, IStakingModule, Versioned {
     /// @param _nodeOperatorId Node Operator id
     /// @param _fromIndex Index of the key, starting with 0
     /// @param _keysCount Number of keys to remove
-    function removeSigningKeys(
-        uint256 _nodeOperatorId,
-        uint256 _fromIndex,
-        uint256 _keysCount
-    ) external {
-        require(_fromIndex < UINT64_MAX, "FROM_INDEX_TOO_LARGE");
-        /// @dev safemath(unit256) checks for overflow on addition, so _keysCount is guaranteed <= UINT64_MAX
-        require(_fromIndex.add(_keysCount) <= UINT64_MAX, "KEYS_COUNT_TOO_LARGE");
-        _removeUnusedSigningKeys(_nodeOperatorId, uint64(_fromIndex), uint64(_keysCount));
+    function removeSigningKeys(uint256 _nodeOperatorId, uint256 _fromIndex, uint256 _keysCount) external {
+        _removeUnusedSigningKeys(_nodeOperatorId, _fromIndex, _keysCount);
     }
 
     /// @notice Removes a validator signing key #`_index` of operator #`_id` from the set of usable keys. Executed on behalf of Node Operator.
     /// @param _nodeOperatorId Node Operator id
     /// @param _index Index of the key, starting with 0
     /// @dev DEPRECATED use removeSigningKeysOperatorBH instead
     function removeSigningKeyOperatorBH(uint256 _nodeOperatorId, uint256 _index) external {
-        require(_index < UINT64_MAX, "INDEX_TOO_LARGE");
-        _removeUnusedSigningKeys(_nodeOperatorId, uint64(_index), 1);
+        _removeUnusedSigningKeys(_nodeOperatorId, _index, 1);
     }
 
     /// @notice Removes an #`_keysCount` of validator signing keys starting from #`_index` of operator #`_id` usable keys. Executed on behalf of Node Operator.
     /// @param _nodeOperatorId Node Operator id
     /// @param _fromIndex Index of the key, starting with 0
     /// @param _keysCount Number of keys to remove
-    function removeSigningKeysOperatorBH(
-        uint256 _nodeOperatorId,
-        uint256 _fromIndex,
-        uint256 _keysCount
-    ) external {
-        require(_fromIndex < UINT64_MAX, "FROM_INDEX_TOO_LARGE");
-        /// @dev safemath(unit256) checks for overflow on addition, so _keysCount is guaranteed <= UINT64_MAX
-        require(_fromIndex.add(_keysCount) <= UINT64_MAX, "KEYS_COUNT_TOO_LARGE");
-        _removeUnusedSigningKeys(_nodeOperatorId, uint64(_fromIndex), uint64(_keysCount));
+    function removeSigningKeysOperatorBH(uint256 _nodeOperatorId, uint256 _fromIndex, uint256 _keysCount) external {
+        _removeUnusedSigningKeys(_nodeOperatorId, _fromIndex, _keysCount);
     }
 
-    function _removeUnusedSigningKeys(
-        uint256 _nodeOperatorId,
-        uint64 _fromIndex,
-        uint64 _keysCount
-    ) internal {
+    function _removeUnusedSigningKeys(uint256 _nodeOperatorId, uint256 _fromIndex, uint256 _keysCount) internal {
         _onlyExistedNodeOperator(_nodeOperatorId);
         _onlyNodeOperatorManager(msg.sender, _nodeOperatorId);
 
+        _requireOutOfRange(_fromIndex < UINT64_MAX);
```

Comment:
Think how to optimimize checks, maybe like this:

require(_fromIndex.add(_keysCount) <= totalSigningKeysCount, "KEY_NOT_FOUND");